Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -135,7 +135,18 @@ extension NSString {
             let index = rangesAndLine.indexOf { NSLocationInRange(offset, $0.byteRange) }
             return index.map {
                 let element = rangesAndLine[$0]
-                return (line: element.line.index, character: offset - element.range.location + 1)
+                
+                let character: Int
+                let content = element.line.content
+                let length = offset - element.byteRange.location + 1
+                if length == element.byteRange.length {
+                    character = content.utf16.count
+                } else {
+                    let endIndex = content.utf8.startIndex.advancedBy(length)
```

Comment:
The content is small enough.
Linting KeychainAccess is still under 4 seconds. :smile:
```
swiftlint lint  3.64s user 0.12s system 90% cpu 4.155 total
```